(5-4)Handling exceptions.py